backend/src/test/java/net/wodie/backend/service/BtoServiceTest.java

```diff
@@ -40,12 +43,402 @@ void updateBtoItemById() {
         assertEquals(initItem1a(), actual);
     }
 
+    @Test
+    void updateBtoItem_statusNew() {
+        //GIVEN
+        BtoItem initItem1a = BtoItem.builder()
+                .id("4")
+                .displayId("t02c1o02")
+                .category("action")
+                .title1("Action No 04")
+                .title2("Second Day")
+                .description("<p>Testeintrag for No4</p>")
+                .status("NEW")
+                .actionOwner("U12")
+                .actionMembers(List.of("U11","U13"))
+                .actionNotMembers(List.of("U15"))
+                .build();
+
+        BtoItem initItem1b = BtoItem.builder()
+                    .id("4")
+                    .displayId("t02c1o02")
+                    .category("action")
+                    .title1("Action No 04")
+                    .title2("Second Day")
+                    .description("<p>Testeintrag for No4</p>")
+                    .status("NEW")
+                    .actionOwner("")
+                    .actionMembers(List.of())
+                    .actionNotMembers(List.of())
+                    .build();
+
+        when(btoRepo.save(initItem1b)).thenReturn(initItem1b);
```

Comment:
Wenn Du A speicherst, sollte er B zurückgeben oder? Hier speicherst Du B und gibtst B zurück.

Reply:
Ich gehe mit A in die Service Methode. In dieser Methode wird der Datensatz zu B verändert und als B an das Repo übergeben. Die Rückgabe vom repo ist dann auch wieder der gespeicherte Datensatz B, was in diesem Test aber gemockt werden muß - daher B in den Mock rein und B wieder raus.